sprints/dashboard/automation.py

```diff
@@ -240,8 +240,10 @@ def check_issue_injected(conn: CustomJira, issue: Issue) -> bool:
         if history.created < cutoff_date:
             break
 
-        if history.items[0].field == "Sprint":
-            return True
+        # Some changelogs can be grouped.
+        for history_item in history.items:
+            if history_item.field == "Sprint":
+                return True
```

Comment:
@Agrendalath, history items with `Sprint` field can contain transitions to `Stretch Goals` or `Backlog`, as well as transitions from old sprints to current sprint.

Is it possible that some ticket with such transitions will be counted as injection here since we're not checking `history_item.toString` value?